apps/src/blockly/googleBlocklyWrapper.ts

```diff
@@ -324,8 +356,7 @@ function initializeBlocklyWrapper(blocklyInstance) {
   blocklyWrapper.blockly_.serialization.registry.unregister('blocks');
   blocklyWrapper.blockly_.serialization.registry.register(
     'blocks',
-    new CdoBlockSerializer(),
-    true /* opt_allowOverrides */
```

Comment:
there is no `opt_allowOverrides` parameter on this function

Reply:
Confirmed